src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -443,5 +489,59 @@ private static byte GetSeverityNumber(LogLevel logLevel)
                 return 1;
         }
     }
+
+    // This method would map the logger category to a table name which only contains alphanumeric values with the following additions:
+    // Any character that is not allowed will be removed.
```

Comment:
What characters are allowed?

Reply:
I thought in line 493, we've already mentioned that only alphanumeric values are allowed?

Reply:
Yes, my bad.